modules/web/server/src/main/resources/logback.xml

```diff
@@ -1,81 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
- Configuration of logging for observe for test
- and production. It will select the file location
- based on the host name
+ Configuration of logging for observe for development and local deployment.
 
- File based logs are rotated daily up to 90 days
+ File based logs are rotated daily up to 90 days.
 
- It supports auto reloading scanning every 60s
+ It supports auto reloading scanning every 60s.
 -->
 <configuration scan="true" scanPeriod="60 seconds">
-  <logger name="org.http4s" level="INFO"/>
-
-  <if condition='property("HOSTNAME").contains("observe")'>
```

Comment:
Since you're removing this (good call) we maybe able to remove janino from the dependencies

Reply:
Good idea done 👍